### ISSUE TITLE
Missing html template for micro-petition:long-petition

### ISSUE THREAD

**Issue by petervojtek** [PowerlineApp/powerline-mobile#147]
@jterps08 , I have a micro-petition:long-petition which renders in following manner in activity newsfeed:

<img width="753" alt="screen shot 2016-07-19 at 07 19 24" src="https://cloud.githubusercontent.com/assets/225506/16939358/3b26d3e6-4d81-11e6-9a59-3483ab4547f6.png">

However when I enter it's detail, I use following template:

<img width="747" alt="screen shot 2016-07-19 at 07 19 49" src="https://cloud.githubusercontent.com/assets/225506/16939363/502b8462-4d81-11e6-86cc-2b93ebfb7a23.png">

I guess we do not use proper html template in detail (the second screenshot) due to the upvote/downvote counts which should not be present for a micro-petition:long-petition.

I do not think we have a html template for a micro-petition:long-petition -- I only guess it should probably show only one button (the same way as in activity newsfeed) -- Sign/Unsign (and maybe count of users who already signed..).

Please correct me if I am wrong.

**Comment by jterps08** [comment]
@petervojtek - It should show a single "sign" button. 

Like this: 
![screenshot_2016-07-19-10-04-19 1](https://cloud.githubusercontent.com/assets/7582457/16957758/b77e557a-4dab-11e6-9f22-758c587b0531.png)

**Also, FYI: For future reference, this is what is supposed to show when a user has different payment options:
![screenshot_2016-07-19-12-43-39](https://cloud.githubusercontent.com/assets/7582457/16959462/373582d2-4db3-11e6-90d6-5fff15cc9c0c.png)

And this is what shows the first time the user gets a payment request (fundraiser/crowdfunder) and taps one of the answers (except $0 Ignore answer) - it will ask for CC information but pre-fill the form with the user's address from their profile.
![screenshot_2016-07-19-12-43-52](https://cloud.githubusercontent.com/assets/7582457/16959605/e42908ce-4db3-11e6-985e-329f76aabcef.png)

And this is what is supposed to show when the "confirmation box" is displayed for a poll/fundraiser
(TBD)
![screenshot_2016-07-19-13-10-01](https://cloud.githubusercontent.com/assets/7582457/16959546/a243e942-4db3-11e6-9c78-5462b3c2796a.png)
or username can be hidden (anonymous donor)
![screenshot_2016-07-19-13-10-09](https://cloud.githubusercontent.com/assets/7582457/16959555/af0a5a62-4db3-11e6-9242-295095bee9e9.png)

If an answer choice is custom (allowing the user to enter his own amount instead of a leader-defined amount), the confirmation box will look like this.
![screenshot_2016-07-19-13-10-15](https://cloud.githubusercontent.com/assets/7582457/16959566/b9a646b6-4db3-11e6-99bd-76f6f7f127ae.png)

After the payment processes successfully, this is what is supposed to shows when a user answers a fundraiser (receipt screen with appropriate info from Stripe*\* transaction):
![screenshot_2016-07-19-13-10-49](https://cloud.githubusercontent.com/assets/7582457/16959579/c4e72752-4db3-11e6-93a1-ea6f6c3ec940.png)

**Comment by petervojtek** [comment]
blocked by #142

**Comment by petervojtek** [comment]
this is blocked by #187

**Comment by petervojtek** [comment]
this is no longer relevant due to changes brought by #187
